### PR TITLE
SPCTR-1801 Fix: Responsive condition option does not save when hide on multiple devices is selected.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-1801

### Screenshots

- https://share.getcloudapp.com/GGuzojwg

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Added to the card.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/extensions/advanced-settings/index.js [hunk 1]
```diff
@@ -187,7 +187,6 @@ const ResponsiveConditionOptions = ( props ) => {
 				onChange={ () =>
 					setAttributes( {
 						UAGHideDesktop: ! attributes.UAGHideDesktop,
-						UAGResponsiveConditions: ! UAGResponsiveConditions
```
Comment (RajkiranBagal) [inline src/blocks/extensions/advanced-settings/index.js:190]: This is needed for old users.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/blocks/extensions/advanced-settings/index.js (hunk 1 above)
Comment (AbhijitNage123) [reply src/blocks/extensions/advanced-settings/index.js:190]: It is causing settings are not saved

Comment (AbhijitNage123) [reply src/blocks/extensions/advanced-settings/index.js:190]: Updated logic. Please verify

Comment (AbhijitNage123) [reply src/blocks/extensions/advanced-settings/index.js:190]: @RajkiranBagal As discussed, I have tested the case and it does not adding any class to block whenever toggle is set to off. Cause we have added condition inside of if again to check is if toggle is true then only add the class to the block. Screencast - https://share.getcloudapp.com/v1uQ2b79

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**